README.md

```diff
@@ -1 +1,37 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# Self Introduction
+Hello. My name is __Soomin Kim__. I am a MA student of _Committee on International Relations_. I was born and raised in South Korea for 23 years and came to the United States to continue my study. I have one younger sister, who is also studying in the Unites States. We love and take care of each other for we are the only family members outside of South Korea. 
+
+# Research Interest
+I have three main research interests.
+
+1. **Nuclear Security**
+
+    Coming from South Korea, I was always exposed to North Korea's nuclear weapon development. Whenever the nuclear missile testing was done, the whole world was reacting to it. I was curious, how can we predict North Korea's behavior in terms of nuclear weapons?, what is the real motivation for them to do so. 
+
+2. **Political Economy**
+
+    World now imposes economic sanctions against North Korea as a warning sign. Does that work? Will North Korea come out to the table and start the talk? If the economic sanctions are not effective, why is that? I think a newly emerging capitalist group, *Donju* dilutes the effectiveness. 
```

Comment:
Minor: perhaps elaborate on Donju? Alternatively, you can include a link - I think readers will be curious to learn about this group!

Reply:
```suggestion
    World now imposes economic sanctions against North Korea as a warning sign. Does that work? Will North Korea come out to the table and start the talk? If the economic sanctions are not effective, why is that? I think a newly emerging capitalist group, (Donju)[link to article] dilutes the effectiveness. 
```